ops/services/app_functions/report_stream_batched_publisher/functions/QueueBatchedReportStreamUploader/index.ts

```diff
@@ -22,10 +22,11 @@ const {
 appInsights.setup();
 const telemetry = appInsights.defaultClient;
 
-const QueueBatchedTestEventPublisher: AzureFunction = async function (
-  context: Context,
+export async function QueueBatchedTestEventPublisher(
+  myTimer: Timer,
```

Comment:
Is the function using this passed-in instance under the hood somehow? Don't see it used anywhere in the file

Reply:
The function execution is being handled/triggered by azure as before. I needed to add both parameters so the function signature would match the type that `app.timer` is expecting.

Reply:
ahh ok, that's what I was guessing. In that case could [we use an underscore to mark that the param isn't being used?](https://stackoverflow.com/questions/27637013/what-is-the-meaning-of-an-underscore-in-javascript-function-parameter#:~:text=A%20single%20underscore%20is%20a,could%20have%20been%20omitted%20entirely.)

Reply:
I'd agree to add the underscore in front, I checked IntelliJ and they even suggest that if you hover over it.
![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/24ee4953-f781-4788-952d-df65797254af)